src/hub.js

```diff
@@ -505,6 +510,10 @@ document.addEventListener("DOMContentLoaded", async () => {
 
   const authChannel = new AuthChannel(store);
   const hubChannel = new HubChannel(store);
+
+  // hacky - we want to access permissions from throughout our A-Frame behaviors and code
+  window.APP.getPermissions = () => hubChannel.permissions;
```

Comment:
FYI, we already have access to the `hubChannel` on `APP` and we also have an A-Frame `permissions` system that exposes a `can` method: https://github.com/mozilla/hubs/blob/master/src/systems/permissions.js
Might be nice to reuse a similar pattern for our other JS code. Maybe just have a `window.APP.permissions.can()` method.

Reply:
didn't know about the system, will refactor